hyper-schema.json

```diff
@@ -1,22 +1,22 @@
 {
-    "$schema": "http://json-schema.org/draft-08/hyper-schema#",
-    "$id": "http://json-schema.org/draft-08/hyper-schema",
+    "$schema": "http://json-schema.org/draft/2019-03/hyper-schema#",
+    "$id": "http://json-schema.org/draft/2019-03/hyper-schema",
+    "$vocabulary": {
+        "https://json-schema.org/draft/2019/03/vocab/core": true,
```

Comment:
should be "2019-03" instead of "2019/03", no? same for "schema.json" below.

Reply:
🤦‍♂️